docs/reference/modules/cache.md

```diff
@@ -0,0 +1,70 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/cache`
+
+**Alias:** `apos.cache`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+The `@apostrophecms/cache` module is a general-purpose cache implementation for improved performance in all areas where results can be retained and reused temporarily. Any number of namespaces can be used to separately maintain data. It is powered by a MongoDB collection.
+
+## Methods
+
+### `async set(namespace, key, value, lifetime)`
+The `set()` method is asynchronous and should be called with an await to insure that any data is cached prior to trying to retrieve it. The method takes four parameters. The first, `namespace`, takes a string and is used to prevent collisions between keys when caching unrelated data. The `key` parameter takes a string and is the identifier that can be used to retrieve the item to be stored. The `value` parameter takes any JSON-friendly value and does not need to be stringified. The optional `lifetime` parameter takes the number of seconds for the value to exist as an integer. If set to zero or unspecified then there won't be an expiration time. However, the cache should not be used for primary storage as it could be cleared at any time, partially cleared to free space, et cetera.
+
+#### Example
+
+```javascript
+…
+const url = `http://api.weatherapi.com/v1/forcast.json?q=${zipCode}`;
```

Comment:
forecast.json

this seems to be a real service... complains of a missing api key after I fix forcast to forecast though